prometheus_all/input.tf

```diff
@@ -27,9 +27,13 @@ variable "alertmanager_slack_url" {
   default     = ""
 }
 variable "alertmanager_slack_channel" {
-  description = "Slack channel for alert notifications"
+  description = "DEPREACTED.  Slack channel for alert notifications.  This is unnecessary if the webhook in alertmanager_slack_url has been assigned to a channel."
```

Comment:
Could we add an output as well to print a message to the users, as in https://stackoverflow.com/questions/66675106/is-there-a-way-to-add-comments-for-teraform-to-display-at-the-end-of-terraform

Reply:
Output doesn't appear in logs unless defined in calling template